src/TensorFlowNET.Core/Operations/math_ops.cs

```diff
@@ -229,6 +229,22 @@ public static Tensor sqrt(Tensor x, string name = null)
 
         public static Tensor mul_no_nan<Tx, Ty>(Tx x, Ty y, string name = null)
             => gen_math_ops.mul_no_nan(x, y, name: name);
+       
+        public static Tensor real(Tensor input, string name = null)
+        {
+            using (var name_ = ops.name_scope(name, "Real", new [] {input}))
```

Comment:
Please use `tf_with()` instead of using for `ops.name_scope()`.

Reply:
Done, for all of the instances of `ops.name_scope()` added originally.